cmd/ulimit_unix_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build darwin || linux
+
+package cmd
+
+import (
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_setUlimit(t *testing.T) {
```

Comment:
Very stupid test, probably can be improved by running on MacOS and also setting ulimit before and ensuring it was changed later